demo/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   ceph:
-    image: quay.io/ceph/daemon:latest-pacific
+    image: quay.io/ceph/daemon:v4.0.13-stable-4.0-nautilus-centos-7-x86_64
```

Comment:
Why was this downgraded to `nautilus`, that version has reached its end-of-life in 2021?  https://docs.ceph.com/en/latest/releases/#active-releases  We should stay on `pacific` because that is closer to what we are running internally.

Also, changes like this should have their own commit, as they are unrelated to the Backstage update.

Reply:
Ah, this was a leftover I think. For some reason the container didn't start due to a missing init file (:question:), so I picked an older version (which worked). I will just remove it